rest/route/github.go

```diff
@@ -352,15 +353,20 @@ func (gh *githubHookApi) tryDequeueCommitQueueItemForPR(pr *github.PullRequest)
 
 	projRef, err := gh.sc.GetProjectWithCommitQueueByOwnerRepoAndBranch(*pr.Base.Repo.Owner.Login, *pr.Base.Repo.Name, *pr.Base.Ref)
 	if err != nil {
-		return errors.Wrapf(err, "can't find projectRef for %s/%s, branch %s", *pr.Base.Repo.Owner.Login, *pr.Base.Repo.Name, *pr.Base.Ref)
+		if adb.ResultsNotFound(err) {
+			return nil
+		}
+		return errors.Wrapf(err, "can't find valid project for %s/%s, branch %s", *pr.Base.Repo.Owner.Login, *pr.Base.Repo.Name, *pr.Base.Ref)
+	}
+
+	if exists, _ := gh.sc.IsItemOnCommitQueue(projRef.Identifier, strconv.Itoa(*pr.Number)); !exists {
```

Comment:
would it work to handle the error for the case when we can't talk to the db? Maybe we could single out a `adb.ResultsNotFound(err)` as being okay, or we could check the projRef has commit queue enabled to be sure that's not the error.

Reply:
Hm well we specifically only look for projects that have a commit queue enabled. If we want to not error for database problems, then we should just return no error if there's no project, regardless of what the error is.

Reply:
Ah, I was wrong.
So should we catch the error?

Reply:
I think this may be best--return an error if there's trouble finding the project, just in case it should still be removed.